include/LCCheating/CheatingParticleIDAlgorithm.h

```diff
@@ -0,0 +1,31 @@
+/**
+ *  @file   LCContent/include/LCCheating/CheatingParticleIDAlgorithm.h
+ *
+ *  @brief  Header file for the cheating particle id algorithm class.
+ * 
+ *  $Log: $
+ */
+#ifndef CHEATING_PARTICLE_ID_ALGORITHM_H
```

Comment:
Typically now qualify the include guard with e.g. LC_CHEATING_PARTICLE_ID_ALGORITHM_H note the 'LC_'